app/views/curation_concerns/base/_attributes.html.erb

```diff
@@ -4,22 +4,22 @@
     <tr><th>Attribute Name</th><th>Values</th></tr>
   </thead>
   <tbody>
-  <%= @presenter.attribute_to_html(:description, 'Abstract') %>
-  <%= @presenter.attribute_to_html(:creator, 'Creator', { catalog_search_link: true }) %>
-  <%= @presenter.attribute_to_html(:contributor, 'Contributors', { catalog_search_link: true }) %>
-  <%= @presenter.attribute_to_html(:subject, "Subject", { catalog_search_link: true }) %>
-  <%= @presenter.attribute_to_html(:publisher, "Publisher") %>
-  <%#= @presenter.attribute_to_html(:bibliographic_citation, "Bibliographic Citation") %>
-  <%#= @presenter.attribute_to_html(:source, "Source") %>
-  <%= @presenter.attribute_to_html(:language, "Language") %>
+  <%= @presenter.attribute_to_html(:description, label: 'Abstract') %>
+  <%= @presenter.attribute_to_html(:creator, catalog_search_link: true ) %>
+  <%= @presenter.attribute_to_html(:contributor, label: 'Contributors', catalog_search_link: true) %>
+  <%= @presenter.attribute_to_html(:subject, catalog_search_link: true) %>
+  <%= @presenter.attribute_to_html(:publisher) %>
+  <%#= @presenter.attribute_to_html(:bibliographic_citation) %>
```

Comment:
Can we make that a ticket and remove these? If we need them back git history should be plenty, yes?